tensorflow/java/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -36,9 +36,22 @@
    * @return a bundle containing the graph and associated session.
    */
   public static SavedModelBundle load(String exportDir, String... tags) {
-    return load(exportDir, tags, null);
+    return load(exportDir, tags, null, null);
   }
-
+   
+    /**
+   * Load a saved model from an export directory. The model that is being loaded should be created
+   * using the <a href="https://www.tensorflow.org/api_docs/python/tf/saved_model">Saved Model
+   * API</a>.
+   *
+   * @param exportDir the directory path containing a saved model.
+   * @param tags the tags identifying the specific metagraphdef to load.
+   * @return a bundle containing the graph and associated session.
+   */
+  public static SavedModelBundle load(String exportDir,byte[] runOptions, byte[] sessionOptions, String... tags) {
```

Comment:
It appears that `sessionOptions` here is just the serialized `ConfigProto` message, not something equivalent to [`TF_SessionOptions`](https://github.com/tensorflow/tensorflow/blob/r1.7/tensorflow/c/c_api.h#L304) which has both a [`ConfigProto`](https://github.com/tensorflow/tensorflow/blob/r1.7/tensorflow/c/c_api.h#L322) and a ["target" address](https://github.com/tensorflow/tensorflow/blob/r1.7/tensorflow/c/c_api.h#L315)  associated with it.

These are a bunch of options, so I'm wondering that instead of having a single function that takes multiple arguments, would it make sense to use the builder pattern for this? So calling code would look something like so:

```java
SavedModelBundle model = SavedModelBundle.loader("/tmp/exportdir")
    .withConfig(configProto)
    .withRunOptions(runOptions)
    .addTag("serve")
    .load();
```

Using something like:

```java
public class SavedModelBundle {
  public final class Loader {
    public Loader withRunOptions(byte[] options) {
      this.runOptions = options;
      return this;
    }
    public Loader withSessionConfig(byte[] config) {
     this.config = config;
     return this;
    }
    public Loader addTag(String tag) ...

    public SavedModelBundle load() { ... }

    private Loader(String exportDir) ...
    // etc
 
   public static Loader loader(String exportDir) { return new Loader(exportDir);  }
   ...
}
```

The private JNI function though could of course just be a flat function that takes all the arguments.

Thoughts?